### PR TITLE
Feature/rails 6

### PR DESCRIPTION
**Pull request by Matthijsy**
### Checklist
- [x] merged database migrations into 1 database migration
- [x] tested database migrations from `origin/staging` (`git checkout staging ; git pull ; bundle exec rails db:reset ; git checkout BRANCH ; bundle exec rails db:migrate`)

### Summary
Let's go rails 6!!

### CODE REVIEW

**Comment by Matthijsy** [comment]
Bullet fails for the following specs:
```
closed_question_answers_controller/update_spec.rb
activities_controller/update_spec.rb
closed_question_answers_controller/create_spec.rb
polls_controller/create_spec.rb
```
I checked in the resources and we do not explicit eager load more fields (and bullet tels us to eager load less). I don't know how to fix it. Probably this loading is done somewhere by jsonapi-resources. If someone could help out it would be great!

**Comment by Matthijsy** [comment]
@cmitz What do you think? Shall we disable bullet for those cases?

**Comment by cmitz** [comment]
Uuh, yes. Maybe on the specific files, because they are tests

File: config/initializers/carrierwave.rb [hunk 1]
```diff
@@ -2,24 +2,17 @@ module UploaderExtensions
...
-
-  CarrierWave::Uploader::Base.descendants.each do |klass|
-    next if klass.anonymous?
-
-    klass.class_eval do
-      prepend UploaderExtensions
-    end
-  end
-end
+# TODO: check that we did need this...
```
Comment (cpbscholten) [inline config/initializers/carrierwave.rb]: Can you check if this is needed before merging?

File: config/initializers/sentry.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Raven.configure do |config|
-  config.dsn = Rails.application.config.x.sentry_dsn
+  # config.dsn = Rails.application.config.x.sentry_dsn
```
Comment (cmitz) [inline config/initializers/sentry.rb]: Fix this plz

**Pull request merged**